lib/airbrake-ruby/payload_truncator.rb

```diff
@@ -106,14 +106,16 @@ def truncate_string(str)
     # For modern Rubies we use UTF-16 as a safe alternative.
     #
     # @param [String] str The string to replace characters
-    # @return [void]
-    # @note This method mutates +str+ for speed
+    # @return [String] a UTF-8 encoded string
+    # @note This method mutates +str+ unless it's frozen,
+    # in which case it creates a duplicate
```

Comment:
According to YARD (the notation we use to generate docs), we should be using two spaces here:

```rb
# @note This method mutates +str+ unless it's frozen,
#   in which case it creates a duplicate
```